pulp_ansible/app/tasks/collections.py

```diff
@@ -422,8 +411,8 @@ class CollectionContentSaver(ContentSaver):
             collection_version = d_content.content
             for d_artifact in d_content.d_artifacts:
                 artifact = d_artifact.artifact
-                with tarfile.open(str(artifact.file.path), "r") as tar:
-                    log.info(_("Reading MANIFEST.json from {path}").format(path=artifact.file.path))
+                with artifact.file.open() as artifact_file, tarfile.open(artifact_file, "r") as tar:
```

Comment:
This needs to be `with artifact.file.open() as artifact_file, tarfile.open(fileobj=artifact_file, mode="r") as tar:`